pkg/k8scontext/context.go

```diff
@@ -130,7 +131,12 @@ func (c *Context) Run(stopChannel chan struct{}, omitCRDs bool, envVariables env
 	var hasSynced []cache.InformerSynced
 
 	if c.informers == nil {
-		return ErrorInformersNotInitialized
+		e := controllererrors.NewError(
+			controllererrors.ErrorInformersNotInitialized,
+			"informers are not initialized",
```

Comment:
The "code" here is kind of duplicate with what your "message" tries to tell, we might just keep code for NewError()?